product_net_weight/models/product_product.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    net_weight = fields.Float(
+        string="Net Weight",
+        digits=dp.get_precision("Stock Weight"),
+        help="Net Weight of the product, container excluded.",
+    )
+
+    weight_price = fields.Float(
+        string="Weight Price",
+        digits=dp.get_precision("Product Price"),
+        compute="_compute_weight_price"
+    )
+
+    # Explicit field, renaming it
+    weight = fields.Float(string="Gross Weight")
```

Comment:
@legalsylvain I would have done something else (as I suppose it's for display purpose):

```suggestion
    gross_weight = fields.Float(related="weight", readonly=False)
```

And make the normal one invisible.

Reply:
Hi. I'm not sure of this proposal. I see the current one lighter. With your design : 

- all the views displaying ``weight`` field will continue to display "Weight" instead of "Gross Weight". 
- to display the gross weight, we'll have to overload each view.
- the models has two identical fields, that is less understandable IMO.

what is the point of your solution ?

Reply:
Yes, that was the bottleneck. Ok for me

Reply:
Talking about bottleneck : it makes sense, on a wine bottle topic...